packages/messenger-widget/src/components/Message/Message.tsx

```diff
@@ -61,14 +57,14 @@ export function Message(props: MessageProps) {
                                   state.uiView.selectedMessageView.messageData
                                       ?.envelop.id === props.envelop.id
                                 ? 'msg-editing-active'
-                                : 'ms-3 background-config-box'
+                                : 'ms-3 own-msg-background'
```

Comment:
Is there a way to make section of complex css modifications more understandable. Its hard to get what is happening here, hence its hard to maintain

Reply:
Later will try to implement optimized css styling